src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/ccd/CaseDetails.java

```diff
@@ -65,10 +61,6 @@ public T getCaseData() {
 
     public LocalDateTime getCreatedDate() {
 
-        if (createdDate == null) {
```

Comment:
Why are you removing this check?

Reply:
I have introduced a preparer class (FeePaymentPreparer) that triggers on start_appeal and at this stage CaseDetails does not have a createdDate or Id.